Source/Core/DolphinQt/HotkeyScheduler.cpp

```diff
@@ -483,6 +483,20 @@ void HotkeyScheduler::Run()
 
       Core::SetIsThrottlerTempDisabled(IsHotkey(HK_TOGGLE_THROTTLE, true));
 
+      if (IsHotkey(HK_TOGGLE_THROTTLE, true) && !Config::Get(Config::MAIN_AUDIO_MUTED) &&
+          Config::Get(Config::MAIN_AUDIO_MUTE_ON_TURBO))
+      {
+        Config::SetCurrent(Config::MAIN_AUDIO_MUTED, true);
+        AudioCommon::UpdateSoundStream(system);
+      }
+      else if (!IsHotkey(HK_TOGGLE_THROTTLE, true) && Config::Get(Config::MAIN_AUDIO_MUTED) &&
```

Comment:
What is the current run check for?

Reply:
The conditional will trigger on every loop without it.  SetCurrent creates the Audio Muted key/value in the CurrentRun layer.  We check to see if it exists along with turbo = off, so we can delete it and refresh the audio stream.